vendor/bat-native-ads/src/bat/ads/internal/processors/behavioral/bandits/epsilon_greedy_bandit_processor.cc

```diff
@@ -156,13 +148,11 @@ void EpsilonGreedyBandit::UpdateArm(const uint64_t reward,
 
   targeting::EpsilonGreedyBanditArmInfo arm = iter->second;
   arm.pulls++;
-  arm.value = arm.value + (1.0 / arm.pulls * (reward - arm.value));
+  arm.value =
+      arm.value + (1.0 / arm.pulls * (static_cast<double>(reward) - arm.value));
```

Comment:
could we add DCHECK that arm.pulls > 0 ?

Reply:
Fixed